public/index.php

```diff
@@ -17,10 +17,7 @@
 
 require __DIR__ . '/../app/AppKernel.php';
 
-$debug       = true;
-$environment = 'dev';
-
-$kernel = new AppKernel($environment, $debug);
+$kernel = new AppKernel('dev', true);
```

Comment:
This index.php hasn't been updated with the kernel changes (namespace, sf4 kernel)

Reply:
I imported the namespace. What else is missing?

Reply:
I removed the call to `loadClassCache`, that at least made it work, but I haven't checked if there is al alternative 🙈